dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -696,6 +698,22 @@ export const CommentLayout = ({ CAPIArticle, NAV, format }: Props) => {
 					/>
 				</ElementContainer>
 
+				{CAPIArticle.storyPackage && (
+					<ElementContainer>
+						<Island deferUntil="visible">
```

Comment:
The 'island' behaviour here is just for hydrating the carousel, right?

Reply:
Yes 😐